modules/twinkleprod.js

```diff
@@ -427,22 +456,31 @@ Twinkle.prod.callback.evaluate = function twinkleprodCallbackEvaluate(e) {
 	Morebits.simpleWindow.setButtonsEnabled(false);
 	Morebits.status.init(form);
 
-	var talk_title = new mw.Title(mw.config.get('wgPageName')).getTalkPage().getPrefixedText();
-	// Talk page templates for PROD-able discussions
-	var blocking_templates = 'Template:Old XfD multi|Template:Old MfD|Template:Oldffdfull|' + // Common prior XfD talk page templates
-		'Template:Oldpuffull|' + // Legacy prior XfD template
-		'Template:Olddelrev|' + // Prior DRV template
-		'Template:Old prod';
-	var query = {
-		'action': 'query',
-		'titles': talk_title,
-		'prop': 'templates',
-		'tltemplates': blocking_templates
-	};
+	Morebits.wiki.actionCompleted.redirect = mw.config.get('wgPageName');
```

Comment:
I don't recall coming across this in my testing. Morebits.wiki.actionCompleted.* is a messy system (basically depends on race conditions) which we probably don't want to use at least in places where taskManager is being used. I would suggest putting 
```js
location.href = mw.util.getUrl(  mw.config.get('wgPageName') ); 
```
in the `execute`'s then block. This method should be more reliable and clearer.

Reply:
I haven't had much time so have been adding/tweaking a bunch of jsdoc stuff since I can do it in 30 second chunks in the morning and, wow, `actionCompleted` is entirely inscrutable to me.  I can try and take a look in September if you don't have a chance, but, oof, there be dragons.